v1/python-sdk/tutorials/automl-with-azureml/forecasting-demand-forecasting-many-models/auto-ml-forecasting-demand-forecasting-many-models.ipynb

```diff
@@ -0,0 +1,1626 @@
+    "\n",
+    "forecasting_parameters = ForecastingParameters(\n",
+    "    time_column_name=time_column_name,\n",
+    "    forecast_horizon=forecast_horizon,\n",
+    "    time_series_id_column_names=time_series_id_column_names,\n",
+    "    cv_step_size=1,\n",
+    "    freq=\"H\",\n",
+    ")\n",
+    "\n",
+    "automl_config = AutoMLConfig(\n",
+    "    task=\"forecasting\",\n",
+    "    debug_log=\"baseline.log\",\n",
+    "    primary_metric=\"normalized_root_mean_squared_error\",\n",
+    "    experiment_timeout_hours=1,\n",
+    "    iterations=1,\n",
+    "    training_data=train_dataset,\n",
+    "    label_column_name=target_column_name,\n",
+    "    compute_target=compute_target,\n",
+    "    enable_early_stopping=True,\n",
+    "    n_cross_validations=2,\n",
+    "    verbosity=logging.INFO,\n",
+    "    max_cores_per_iteration=-1,\n",
+    "    enable_dnn=False,\n",
+    "    allowed_models=[\"Naive\"],\n",
+    "    forecasting_parameters=forecasting_parameters,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(f\"Is the baseline model trained? {IS_BASE_MODEL_TRAINED}\\n---\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "IS_BASE_MODEL_TRAINED = False"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "if not IS_BASE_MODEL_TRAINED:\n",
+    "    remote_base_run = experiment.submit(automl_config, show_output=False)\n",
+    "else:\n",
+    "    from azureml.train.automl.run import AutoMLRun\n",
+    "\n",
+    "    BASE_RUN_ID = \"<Replace with the run ID used for training the baseline model>\"\n",
+    "    # during the initial training run copy-paste the run id to be utilized later if needed.\n",
+    "    remote_base_run = AutoMLRun(experiment=experiment, run_id=BASE_RUN_ID)\n",
+    "    print(f\"Using previously trained model. Run ID: {BASE_RUN_ID}\\n---\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if not IS_BASE_MODEL_TRAINED:\n",
+    "    remote_base_run.wait_for_completion(show_output=False)\n",
+    "    IS_BASE_MODEL_TRAINED = True"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 6. Test set inference "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 6.1 Many models inferences\n",
+    "\n",
+    "We create an output folder which will be used to save the output of our experiments."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# create an output folder\n",
+    "OUTPUT_DIR = os.path.join(os.getcwd(), \"forecast_output\")\n",
```

Comment:
We can just say `OUTPUT_DIR = \"forecast_output\"`

Reply:
done